src/py42/services/storage/exfiltrateddata.py

```diff
@@ -26,8 +26,9 @@ def get_download_token(self, event_id, device_id, file_path, timestamp):
         params = "deviceUid={}&eventId={}&filePath={}&versionTimestamp={}"
         params = params.format(device_id, event_id, quote(file_path), timestamp)
         resource = "file-download-token"
+        headers = {"Accept": "*/*"}
```

Comment:
was this header not being present breaking anything before?

Reply:
not in the production sense, but the way the mock server works is that the response type _must_ be compatible with the request's `Accept` headers, so this is needed for the integration tests to work.